multipaz-compose/src/commonMain/kotlin/org/multipaz/compose/Util.kt

```diff
@@ -159,4 +159,10 @@ expect fun rememberUiBoundCoroutineScope(
     getContext: @DisallowComposableCalls () -> CoroutineContext = {
         EmptyCoroutineContext
     }
-): CoroutineScope
\ No newline at end of file
+): CoroutineScope
+
+internal expect fun overLayWithText(
+    imageBytes: ByteArray,
+    primaryText: String?,
+    secondaryText: String?,
+): ByteArray
```

Comment:
This seems specific to DocumentModel so suggest to move there (and introduce DocumentModel.android.kt and DocumentModel.ios.kt for the specializations).

I also think it probably's easier to understand what's going on and more performant if you change it to take an ImageBitmap and return an ImageBitmap.